src/app/components/elements/cards/QuestionCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { useExpandContent } from "../markup/portals/Tables";
+import { useStatefulElementRef } from "../markup/portals/utils";
+import { Button, Card, CardBody, Row } from "reactstrap";
+import classnames from "classnames";
+import { IsaacContent } from "../../content/IsaacContent";
+import { IsaacQuestionPageDTO } from "../../../../IsaacApiTypes";
+
+const GoToQuestionFinder = () => (
+  <>
+    <h2>All done! Want more questions?</h2>
+    <Row className="w-75 mx-auto mt-5 mb-3">
+      <Button href="/gameboards/new" className="btn-block btn-lg btn-primary">
+        Go to Question Finder
+      </Button>
+    </Row>
+  </>
+);
+interface QuestionCardProps {
+  setExpanded: (expanded: boolean) => void;
+  questionData: IsaacQuestionPageDTO[];
+}
+
+const QuestionCard = ({ setExpanded, questionData }: QuestionCardProps) => {
+  const [expandRef, updateExpandRef] = useStatefulElementRef<HTMLDivElement>();
+  const { expandButton, outerClasses, expanded } = useExpandContent(true, expandRef);
+  const [question, setQuestion] = useState<number>(0);
+  const isMoreQuestions = question < 5;
+
+  useEffect(() => {
+    setExpanded(expanded);
+  }, [expanded, setExpanded]);
+
+  return (
+    <div className={!expanded ? "question-tile" : ""} data-testid="question-tile">
+      <Row className="m-0 d-flex justify-content-between">
+        <h3 className="font-weight-normal m-0 align-self-baseline">Quick question!</h3>
+        {isMoreQuestions && (
+          <Button className="next-question bg-transparent border-0 btn-link" onClick={() => setQuestion(question + 1)}>
+            Next question
+          </Button>
+        )}
+      </Row>
+      <Card
+        className={classnames(outerClasses, expanded ? "random-question-panel" : "mt-2 pb-2")}
+        data-testid="question-content-card"
+        style={expanded ? { maxHeight: "560px" } : { maxHeight: "450px" }}
+      >
+        <CardBody className="p-3">
+          <div
+            id="question-body"
+            ref={updateExpandRef}
+            style={expanded ? { maxHeight: "500px" } : { maxHeight: "400px" }}
+            className="overflow-auto hidden-scrollbar"
+          >
+            {isMoreQuestions ? <IsaacContent doc={questionData[question]} /> : <GoToQuestionFinder />}
+          </div>
+          {expanded ? expandButton : isMoreQuestions && expandButton}
```

Comment:
Maybe this is just me, but I find this a bit hard to read. Isn't this equivalent to:
```typescript
{expanded || isMoreQuestions ? expandButton : null}
```

Reply:
Perhaps this instead?

`{(expanded || isMoreQuestions) && expandButton}`

Reply:
I tend to never use `condition && theThingThatWeWantToDisplay` because it relies on the fact that in JS, the `theThingThatWeWantToDisplay` will be returned if `condition` is truthy, but this is not how it works in most languages. Normally, in other languages, the result of this expression is a boolean. So I find it less readable. But if the way you have suggested is common practice in JS and any JS developer would understand it without problems, then go ahead. I just tend to error on the side of caution and write code as readable as possible.

Reply:
Yeah it's perfectly understandable in JS. I guess I have the (dubious!) advantage of having worked almost exclusively in JS so this feels like standard practise for me - I'll make sure to be aware of it in other languages though!